source/Ship.h

```diff
@@ -310,6 +310,9 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	int Crew() const;
 	int RequiredCrew() const;
 	void AddCrew(int count);
+	// Get this ship's crew equivalent value, depending on its crew and its
+	// "crew equivalent" attribute that is mostly used for drones.
```

Comment:
How about this instead?
```suggestion
	// Get the reputational value of this ship's crew, which depends
	// on its crew size and "crew equivalent" attribute.
```